### PR TITLE
[MODULAR] Rebinds Hypospray Controls

### PR DESCRIPTION
**Pull request by softcerv**
## About The Pull Request
Remaps the Hypospray controls to be more responsive and use the Right mouse button.

- Injecting is done by left clicking a patient, spraying is done by right clicking.
  - Because of this change, the button to swap between spraying and injecting has been removed
- You can now left click on a Hypospray or use it in hand to change the transfer amount
  - Alternatively you can now Right click to decrease the transfer amount. 
- Ejecting a vial is now mapped to using an empty hand on a Hypospray loaded with a vial, similar to ballsitic guns work.

Here is a video of the changes in action:

https://user-images.githubusercontent.com/68373373/142527012-b3157f30-1905-4315-a5aa-45b244e9aaa8.mp4

https://user-images.githubusercontent.com/68373373/142581784-5ddf0690-dadd-455c-b42e-657c023c7e46.mp4

## How This Contributes To The Skyrat Roleplay Experience
This should make the Hypospray feel more responsive to use, making it a better tool for doctors.

## Changelog

:cl:
qol: the controls on the Hypospray have been rebinded to take advantage of the right mouse button, please check  :https://github.com/Skyrat-SS13/Skyrat-tg/pull/9558 for details
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
I'm not sure right mouse to eject is intuitive. It should, in my opinion, be done through an attack with an empty hand, à la guns with their magazines.

**Comment by softcerv** [comment]
> 
> 
> I'm not sure right mouse to eject is intuitive. It should, in my opinion, be done through an attack with an empty hand, à la guns with their magazines.

I could possibly look into that.

File: modular_skyrat/modules/hyposprays/code/hyposprays_II.dm [hunk 1]
```diff
@@ -218,31 +225,24 @@
...
-				mode = HYPO_SPRAY
-				to_chat(user, "[src] is now set to spray contents on application.")
-		return TRUE
+	else
+		. = ..()
 
 /obj/item/hypospray/mkii/examine(mob/user)
 	. = ..()
-	. += span_notice("<b>Ctrl-Click</b> it to toggle its mode from spraying to injecting and vice versa.")
+	. += span_notice("<b>Left Click</b> on patients to inject, <b>Right Click</b> to spray.")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/hyposprays/code/hyposprays_II.dm]:
```suggestion
	. += span_notice("<b>Left-Click</b> on patients to inject, <b>Right-Click</b> to spray.")
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Tiny nitpick.

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**